cms/djangoapps/contentstore/tasks.py

```diff
@@ -14,6 +15,11 @@
 from contentstore.utils import initialize_permissions
 from opaque_keys.edx.keys import CourseKey
 
+from contentstore.courseware_index import CoursewareSearchIndexer, SearchIndexingError
+
+LOGGER = get_task_logger(__name__)
```

Comment:
The latest pylint requires that globals have UPPERCASE names, not constants - it's either this or add a pylint ignore flag.